File: Perl/fizzbuzz1.pl
$LIMIT = 100;

for ($i = 1; $i <= $LIMIT; $i++) {
	if ($i % 15 == 0) {
		print "FizzBuzz";
	} elsif ($i % 3 == 0) {
		print "Fizz";
	} elsif ($i % 5 == 0) {
		print "Buzz";
	} else {
		print $i;
	}
	if ($i < $LIMIT) {
		print ", ";
	}
}

print "\n";

File: Ruby/fizzbuzz1.rb
LIMIT = 100

for i in 1..LIMIT
  if (i % 15).zero?
    print 'FizzBuzz'
  elsif (i % 3).zero?
    print 'Fizz'
  elsif (i % 5).zero?
    print 'Buzz'
  else
    print i
  end
  print ', ' if i < LIMIT
end

puts
